### PR TITLE
UI bugs + localization

### LINKED ISSUE #164: Language localization everywhere

### LINKED ISSUE #213: Logout button is not aligned from the right equally as the home page from the left

### LINKED ISSUE #214: Create button is not in the center on phone mode

### LINKED ISSUE #216: No plios placeholder needs to be fixed

### LINKED ISSUE #217: Make URL tracking tighter configurable

### LINKED ISSUE #218: Phone number invalid is shown even without any phone input

### LINKED ISSUE #220: video link shows invalid without any input added

### PR DESCRIPTION
**Pull request by dalmia**
Fixes #164 
Fixes #214 
Fixes #213 
Fixes #216 
Fixes #217
Fixes #218
Fixes #220

## Summary
- localization strings everywhere
- logout button positioning
- create button centered
- new placeholder for no plios
- don't show input validation if nothing has been entered.
- editor: video link invalid shown only if there is any input added

## Test Plan
Tested locally

### CODE REVIEW

File: src/assets/locales/en.js [hunk 1]
```diff
@@ -12,45 +12,227 @@ export default {
...
-    tutorial: {
-      mcq: "Choose any one",
+    fullscreen: {
+      enter: "Go Fullscreen",
+      exit: "Exit Fullscreen",
     },
   },
   error: {
     404: "Page Not Found",
+    create_plio: "Error creating Plio",
```
Comment (deepansh96) [inline src/assets/locales/en.js:157]: Just an opinion!
```suggestion
    create_plio: "Could not create Plio. Please try again",
```

File: src/assets/locales/en.js [hunk 2]
```diff
@@ -12,45 +12,227 @@ export default {
...
+      status: {
+        published: "This plio has been published and is publicly accessible",
+        draft:
+          "This plio is currently in draft mode and only accessible to you. To make it publicly accessible, publish the plio",
+      },
+      video_input: {
+        published: "You cannot edit the video URL in a published plio",
+      },
+      add_item: {
+        published: "Adding new questions in a published plio is not allowed",
```
Comment (deepansh96) [inline src/assets/locales/en.js:197]:
```suggestion
        published: "You cannot edit the video URL in a published plio",
      },
      add_item: {
        published: "You cannot add new questions in a published plio.",
```

**Review by deepansh96 — CHANGES_REQUESTED** [review]

**Comment by deepansh96** [comment]
- "Untitiled" is left out - should also be in hindi (image 1)
(IMAGE 1)
Screenshot 2021-05-03 at 9 15 30 AM

- I guess this got missed out. Line no 421 in `ItemEditor.vue` (image 2)
(IMAGE 2)
Screenshot 2021-05-03 at 12 48 53 PM

- Something is causing the red cross button to become extremely small on smaller screens (image 3), which was not the case before (image 4)
(IMAGE 3)
Screenshot 2021-05-03 at 1 12 31 PM
(IMAGE 4)
Screenshot 2021-05-03 at 1 13 40 PM

File: src/components/Items/ListItems/PlioListItem.vue [hunk 3]
```diff
@@ -8,7 +8,7 @@
```
[the remark this answers is not in the document]
Comment (dalmia) [reply src/components/Items/ListItems/PlioListItem.vue:8]: move to `Home/Table` - both this and `Table.vue`

File: src/components/Items/Question/Header.vue [hunk 4]
```diff
@@ -1,5 +1,5 @@
 <template>
-  <div class="flex w-full bg-white justify-end p-1 md:p-3 pr-3 py-3">
+  <div class="flex w-full bg-white justify-end p-1">
```
[the remark this answers is not in the document]
Comment (dalmia) [reply src/components/Items/Question/Header.vue:2]: add right padding

**Review by deepansh96 — APPROVED** [review]

**Pull request merged**